site/src/pages/[shortname]/blocks/[block-slug].page.tsx

```diff
@@ -251,11 +251,22 @@ const BlockPage: NextPage<BlockPageProps> = ({
 
   return (
     <>
-      <Head>
-        <title>
-          {`Block Protocol – ${blockMetadata.displayName} Block by ${shortname}`}
-        </title>
-      </Head>
+      <NextSeo
+        title={`Block Protocol – ${blockMetadata.displayName} Block by ${shortname}`}
+        description={
+          blockMetadata.description ||
+          "Check out this open-source block on Block Hub"
```

Comment:
This copy needs a pass

Reply:
Not sure if I understand. Should we change it to something else?